src/reports/components/instance-details.scss

```diff
@@ -61,8 +50,6 @@
     background: $neutral-0;
     display: table;
     table-layout: fixed;
-    @include fill-available-width;
```

Comment:
Was it intentional to remove this entirely rather than replacing it with `width: fill-available;`?